api/api.proto

```diff
@@ -987,3 +987,121 @@ message ClusterAlertEraseRequest {
 
 message ClusterAlertEraseResponse {
 }
+
+// CreateClusterPairRequest
+// swagger:model
+message CreateClusterPairRequest {
```

Comment:
Change to `ClusterPairXXXX` style to be compatible with the SDK. So this should be `ClusterPairCreateRequest` and so on.